_plugins/simple_search_filter.rb

```diff
@@ -0,0 +1,19 @@
+module Jekyll
+  module CharFilter
+    def remove_chars(input)
+      input.gsub! '\\','&#92;'
+      input.gsub! /\t/, '    '
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.